### ISSUE TITLE
Gather companies participating in Hacktoberfest 2024

### ISSUE THREAD

**Issue by crweiner** [crweiner/hacktoberfest-swag-list#876]
Hello you beautiful nerds! 🤓  It's that amazing time of year again! With Hacktoberfest right around the corner, it's time to start gathering the list of companies who celebrate Hacktoberfest, purposely want to draw more contributors to their projects, and will reward meaningful contributions with sweet, sweet swag.

I am working on the [prepare-for-2024](https://github.com/crweiner/hacktoberfest-swag-list/tree/prepare-for-2024) branch to get everything setup, but I need your help to add companies to the list once more.

Until Hacktoberfest kicks off formally, let's use this GH Issue to keep track of the companies who have announced their participation. After October 1st, we will solicit adding companies to the repo via their own GH Issue, [just like before](https://github.com/crweiner/hacktoberfest-swag-list/issues?q=is%3Aissue+is%3Aclosed).

I hope everyone is ready for a great Hacktoberfest 2024! 🥇

**Comment by nooras** [comment]
Hey @crweiner feel free to ping me if you need any help.

**Comment by crweiner** [comment]
Appwrite will be participating this year by hosting a hackathon and giving out prizes to the top 3 teams: https://appwrite.io/blog/post/appwrite-hacktoberfest-hackathon-2024

**Comment by crweiner** [comment]
Servicenow will also be participating: https://www.servicenow.com/community/developer-advocate-blog/servicenow-hacktoberfest-2024/ba-p/3052690

> From ServiceNow Community

> * Community Badge. A Community Badge for anyone who "completes" Hacktoberfest (4 accepted Pull Requests to eligible projects)
> * ServiceNow Merch. Top contributors, high quality contributors, early contributors, and a random selection of contributors will be given codes to the ServiceNow store to redeem ServiceNow merch (while supplies last)
> * Bragging rights! The main ServiceNow Hacktoberfest repo (https://github.com/ServiceNowDevProgram/Hacktoberfest) has a leaderboard to track all of our top contributors for this year and all-time.

**Comment by crweiner** [comment]
devICT is returning: https://hacktoberfest.devict.org/
> Submit two pull requests to qualifying Wichita projects during the month of October and receive stickers from us!

**Comment by crweiner** [comment]
Hyperswitch is also offering unique swag this year: https://dev.to/hyperswitchio/hacktoberfest-2024-with-hyperswitch-47dk

> Get one PR merged to earn exclusive Hyperswitch swag.

> Get 3 or more PRs merged to receive a special swag kit from Hyperswitch.

**Comment by crweiner** [comment]
Turbot is getting into the Hacktoberfest spirit! https://turbot.com/blog/2024/09/hacktoberfest-2024

> Any special swag from Turbot?
Yes! In addition to the DigitalOcean event, Turbot is adding our own swag for contributions to our projects. We will offer unique Turbot stickers and t-shirts for no-code, low-code and high-code contributions.

**Comment by crweiner** [comment]
MindsDB is also back to enjoy the Hacktoberfest fun: https://mindsdb.com/hacktoberfest

> Collect credits by completing GitHub issues and redeem them for prizes!
> For every 100 credits earned, you will receive one entry into the drawing to win one of three high end Razer™ Blade laptops.
> Redeem credits for MindsDB swag.

**Comment by vimode** [comment]
Hey @crweiner  👋  Hope you are doing good! I will try to be here to help this year too!

Layer5 https://layer5.io/programs/hacktoberfest
> Our Hacktoberfest theme for 2024 will focus on extending Meshery models.
> Earn a Hacktoberfest Contributor badge by contributing to Layer5 projects during this Hacktoberfest.

**Comment by vimode** [comment]
Authtoberfest 2024 - https://auth0.com/blog/announcing-authtoberfest-2024/

> Contribute to any of the projects in the Identity and Security space shortlisted by us. The first 100 valid submissions will get the T-shirt.
> A GitHub user with two accepted/merged PRs sending us the confirmation emails and the Hacktoberfest badge will be considered a valid submission.
> Visit https://authtoberfest.io/ to learn more about the event and how you can participate.

**Comment by vimode** [comment]
tscircuit https://blog.tscircuit.com/p/build-electronics-with-typescript

 > 5 contributions to win a hat
 > at least 20 contributions, at least 5 major contributions
 > We currently can only ship to the U.S. but we are working on shipping internationally.

**Comment by vimode** [comment]
hey @crweiner are Umbraco not to be included this year too? 

Umrabco https://umbraco.com/blog/join-us-for-umbraco-hacktoberfest-2024/

> If you get your 2 contributions accepted, you’re automatically entered into a raffle to get a free [official Umbraco training](https://umbraco.com/training/) of your choice. We’ll raffle off training to five lucky, randomly selected, participants. As always, you can also apply for swag or donation of trees. 
> There will be swag again this year! We’ll reveal what exactly you can earn later on in the month. 🎁
> As always, if you don’t want swag, then we’ll plant a bunch of trees for you in your name for the upcoming year. 🌳🌲

**Comment by vimode** [comment]
Instill AI - https://dev.to/shihchun-h/announcing-instill-ais-tentaculous-hacktoberfest-2024-3991

>  Any participant whose contribution is accepted by Instill AI will earn a swag package, which includes Instill Physical Stickers, and an Instill Bottle or Mug.
> Participants who rank in the top five based on points will receive a $50 Amazon Gift Card.

**Comment by vimode** [comment]
DocsGPT - https://github.com/arc53/DocsGPT/blob/main/HACKTOBERFEST.md

> Top 50 contributors will receive a special T-shirt

**Comment by vimode** [comment]
Taipy - https://assorted-son-815.notion.site/HacktoberFest-2024-with-Taipy-2a5032a3f01642709e88ffaa5d0d169e

> Contribute to issues from easy to advanced difficulty to earn points to win Swag (Stickers, bottle, hoodie, mouse pad, bag)

**Comment by vimode** [comment]
Flyte - https://github.com/flyteorg/flyte/issues/5783

> Code and content contributions accepted. Blog post with an overview of the prizes (coming soon)

**Comment by vimode** [comment]
Gradle  Plugin for Jenkins - https://community.gradle.org/events/hacktoberfest/2024/

> sending custom Gradle goodies for those who do substantial contributions to the Gradle ecosystem. (hats, stickers)

**Comment by vimode** [comment]
Configu - https://github.com/configu/configu/issues/465

>  Code and Non-Code Contributions accepted. 
>  1 Valid Pull Request: Receive an exclusive Configu Beer Glass. 
>  2 Valid Pull Requests: Secure a LIMITED-EDITION Configu Hacktoberfest T-shirt. 
>   3+ Valid Pull Requests: Get both the Configu Beer Glass and the LIMITED-EDITION Hacktoberfest T-shirt!

**Comment by vimode** [comment]
TBD - https://github.com/TBD54566975/developer.tbd.website/issues/1680

> The top 10 contributors with the most points will snag custom swag with this year's exclusive TBD x Hacktoberfest 2024 design.

**Comment by vimode** [comment]
Reflex - https://github.com/reflex-dev/hacktoberfest

> Build apps using Reflex or Wrapping custom components to receive cash rewards, custom-branded swag,

**Comment by vimode** [comment]
Modtoberfest - https://modtoberfest.com/

> Those who complete our challenge will be rewarded with a prize pack that includes awesome stickers designed by the sponsors of our event! We will also be planting a tree for each person who completes the event

**Comment by vimode** [comment]
Devfest AI - https://devfest.ai/#swag

> Participate in exciting challenges and receive unique goodies from our sponsors.

**Comment by vimode** [comment]
Crawlee for Python - https://github.com/apify/crawlee-python/issues/555

>1-2 Accepted Pull Request: Crawlee Exclusive Sticker Sheet.
>2 or more Accepted Pull Requests: Crawlee Exclusive Hoodie.

**Comment by vimode** [comment]
Cloudinary - https://cloudinary.com/blog/hacktoberfest-open-source-celebration-cloudinary

> Like last year, we’re giving away 50 special swag packs, including a limited edition 2024 Hacktoberfest T-shirt, stickers, and the beloved Cloudinary unicorn for any valid submission.

**Comment by vimode** [comment]
Animata - https://animata.design/blog/hacktoberfest-2024

> We’re offering a $50 reward to one lucky contributor based on a raffle system.

**Comment by crweiner** [comment]
So happy to see you back again, @vimode! Happy Hacktoberfest!

> Hey @crweiner 👋 Hope you are doing good! I will try to be here to help this year too!
Can I start contributing to the [prepare-for-2024](https://github.com/crweiner/hacktoberfest-swag-list/tree/prepare-for-2024?rgh-link-date=2024-09-26T17%3A39%3A24Z) branch? or should I wait for it to be merged with main and then contribute?

> Thank you for the support last year!

Now that Hacktoberfest is here, please feel free to submit a PR against the `master` branch!

**Comment by crweiner** [comment]
> hey @crweiner are Umbraco not to be included this year too?

> Umrabco https://umbraco.com/blog/join-us-for-umbraco-hacktoberfest-2024/

Correct, let's keep them off the list. I have updated the blocklist title to reflect this for 2024 as well: https://github.com/crweiner/hacktoberfest-swag-list/issues/256

**Comment by crweiner** [comment]
@vimode, I've turned every project you found above into it's own Issue for better tracking, now that Hacktoberfest is here! I'll close this prep Issue now, and we can handle adding new companies/projects to the list just like last year. 😄
